lib/web/mage/adminhtml/tools.js

```diff
@@ -364,6 +364,7 @@ var Fieldset = {
 var Base64 = {
     // private property
     _keyStr: 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/=',
+    _notBase64 : /[^A-Z0-9+\/=]/i,
```

Comment:
More precise regexp may be used to verify is string looks like base64 encoded or not:
```
(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)
```